src/user/views/user_views.py

```diff
@@ -803,6 +804,45 @@ def sift_check_user_content(self, request):
             raise Exception("Sift verification signature mismatch")
 
 
+@api_view([RequestMethods.GET])
```

Comment:
@le-sun will this speed up API requests 🤔 ?

Reply:
To sum up what we went over during the meeting:

Because class based views has a lot of inheritance, it adds some overhead when processing a request. Function based views are relatively barebones, so there is a slight performance boost.

CBVs are nice to have because of inheritance. It provides a lot of default methods that we don't have to implement ourselves when creating a view.
FBVs are simple, which means it should be used for simple endpoints. Adding more complex logic in FBVs is redundant since CBVs provide a lot of utility for complex logic.